core/src/CMakeLists.txt

```diff
@@ -26,7 +26,9 @@ IF (NOT desul_FOUND)
     SET(DESUL_ATOMICS_ENABLE_OPENMP ON) # not a typo Kokkos OpenMPTarget -> Desul OpenMP
   ENDIF()
   IF(KOKKOS_ENABLE_OPENACC)
-    SET(DESUL_ATOMICS_ENABLE_OPENACC ON)
+    IF(KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
```

Comment:
You want a comment, something greppable so you remember to adjust whenever Clacc catches up?

Reply:
Added a comment.